JDI.Light/JDI.Light.Tests/Utils/FileLogger.cs

```diff
@@ -10,7 +10,7 @@ namespace JDI.Light.Tests.Utils
 {
     public class FileLogger : ILogger
     {
-        public const string LogFilePath = @"C:\temp\jdi.txt";
+        public static string LogFilePath = @"C:\temp\jdi.txt";
```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5848342)